### DISCUSSION TITLE
Improvements to default behaviors of the Project Manager #5768

### DISCUSSION THREAD

**Discussion by GemExperience** [godotengine/godot-proposals#5774] [General Discussions]

**Comment by Zireael07** [comment]
Calling a core contributor and the main person responsible for the proposals repo "idiotic" is not a good way to get things done.

**Reply by Zireael07** [reply]
Replying to mysef: remembering the last open folder is, however, a good idea.

**Reply by GemExperience** [reply]
Closing an issue for the reason given, pointing out irrelevant personal opinions and you as well is not appreciated nor good way to get anything done either. I have many other good ideas that I could contribute with. Not all of us are programmers ! And just before this turns into a soap opera, as explained above, no personal attacks here. All I am doing is giving suggestions and pointing out flaws in closing a perfectly viable issue ie suggestion that could be worked on, instead we are left with comments like yours that steer away from the topic at hand. Your so called core contributor should not close topics like that nor demand one to have a degree in programming to bring up simple issues like these. Both of my ideas are good. Being able to change or correct the folder path in a Project Manager is a perfectly good idea to look into also.

**Reply by YuriSizov** [reply]
Your proposal was closed because it was a duplicate, first and foremost. This means that there is already another tracked issue that is perfectly suitable for all the relevant discussion. Having multiple issues for the same suggestion is not useful, divides attention of users, and creates bloat and unnecessary load for the maintainers. We close duplicates, linking the relevant issue, all the time. So does everyone else, both in open source and in corporate worlds, as this is a normal practice in project management.

As for the rest of the issue, the rules of engagement are clearly stated in the readme document of the proposal repository. Proposal *issues* are used for actionable things, containing implementation detail and concrete suggestion on how to solve the problem. If you only have a general suggestion but cannot work on the implementation or at least theorize on how it should be implemented, then you should use this tab, proposal *discussions*. It exists specifically to accommodate non-programmers or junior developers, or just anyone really who wants to shoot ideas before committing to a solution. And you were politely directed here, despite your original message containing rude remarks such as "It's your job".

**Comment by Mickeon** [comment]
Do note that blanking out your feature requests for every other user casually reading them makes discussing about the (rightfully) requested features all the more confusing, and is not a good way to get things done, either.

- For any user that does not know it is possible to see the edited comment history, the feature request is the same as https://github.com/godotengine/godot-proposals/issues/5768, which was marked as a duplicate of https://github.com/godotengine/godot-proposals/issues/1312 because the latter comes with a more appropriate explanation to the same issue, as well as the solution.

It's only a matter of time before someone implements the feature. It just may not be on everyone's head right now, as there are more pressuring issues at hand, such as bug-fixes for the ever-so-close 4.0 stable.

**Reply by GemExperience** [reply]
Do note that comments like yours is not a good way to get things gone either.

Closing a rightfully requested features is exactly what I pointed out above. Don't shoot the messenger. Unfortunately the topic was trashed by Zireael07's off topic commentary, and now I have to deal with your useless commentary as well.

Fact is, I gave TWO perfectly reasonable suggestions:

1: Fix the default path. 

2. Let users fix the path for each project IN THE PROJECT MANAGER

Lesson be learned, respect is earned. Get to work.

**Comment by timothyqiu** [comment]
You can change the default folder in Editor Settings: `filesystem/directories/default_project_path`.

Although I also want to edit the path of a moved project occasionally, I think for your use case, after moving lots of projects to another root folder, it'll be more efficient to Remove Missing and Scan the new root folder.

p.s. If you are comfortable with markdown, I think you can replace all-caps with `**content**` (for **content**) or `*content*` (for *content*), so that other people won't mistake you for yelling.

**Comment by akien-mga** [comment]
Locking as the OP is engaging in disruptive behavior which breaches Godot's [Code of Conduct](https://godotengine.org/code-of-conduct), and was therefore banned for 7 days from interacting on GitHub.

@GemExperience Acting entitled and harassing contributors is not allowed. If you decide to come back after your 7-day ban, please stay respectful and constructive in your posts.

As others pointed out the linked proposal was a duplicate of https://github.com/godotengine/godot-proposals/issues/1312, so further discussion (about the proposed change) should happen there.
